src/pages/Calendar/RunMenu.vue

```diff
@@ -16,7 +17,15 @@ export default {
     type: { type: String, required: false, default: 'task-run' },
     active: { type: Boolean, required: true }
   },
-  computed: {},
+  computed: {
+    ...mapGetters('api', ['isCloud']),
+    showScheduleBanner() {
+      if (this.run.state === 'Scheduled' && this.active && this.isCloud) {
+        return true
+      }
+      return false
```

Comment:
this can be written as just `return this.run.state === 'Scheduled' && this.active && this.isCloud`